homeassistant/helpers/template.py

```diff
@@ -865,6 +867,13 @@ def expand(hass: HomeAssistantType, *args: Any) -> Iterable[State]:
     return sorted(found.values(), key=lambda a: a.entity_id)
 
 
+def device_entities(hass: HomeAssistantType, device_id: str) -> Iterable[str]:
+    """Get entity ids for entities tied to a device."""
+    entity_reg = entity_registry.async_get(hass)
+    entries = entity_registry.async_entries_for_device(entity_reg, device_id)
```

Comment:
This will omit disabled entities. Should we return all the entity IDs here instead?

Reply:
Is there any means via a template to determine if a given entity is disabled?  So - if I receive a list of entities, and some might be disabled, can I then do something to determine which ones are enabled or disabled?

Reply:
I think that it's fine to omit disabled entities, as they will not be available as a state anyway.